http-filter-example/envoy-sample-config.yaml

```diff
@@ -45,7 +45,7 @@ static_resources:
             typed_config:
               "@type": type.googleapis.com/sample.Decoder
               key: header-processing
-              val: "http-request set-header x-forwarded-proto https\n 
+              val: "http-request set-header x-forwarded-proto https\n
```

Comment:
This could be a mult-line string in YAML with `|`, to avoid the quotes and `\n`:

```
val: | 
  http-request set-header x-forwarded-proto https
  http-request set-header mock_key mock_val1 mock_val2
```

Here's an example: https://github.com/DataDog/k8s-resources/blob/master/k8s/ingress-haproxy/templates/configmap_envoy.yaml#L17